src/Text/KirbyTag.php

```diff
@@ -44,8 +45,7 @@ public function __construct(
 			$type = static::$aliases[$type];
 		}
 
-		$kirby    = $data['kirby'] ?? App::instance();
-		$defaults = $kirby->option('kirbytext.' . $type, []);
+		$defaults = $this->kirby()->option('kirbytext.' . $type, []);
```

Comment:
The `$data` prop is set below, so it isn't available when `$this->kirby()` is called from here.